src/components/Trade/Trade.tsx

```diff
@@ -36,12 +44,17 @@ export const Trade = () => {
       fiatAmount: undefined
     }
   })
-
+  const match = useParams<MatchParams>()
+  const assetId = !match.accountId
+    ? `${match?.chainId}/${match?.assetSubId}`
+    : `${decodeURIComponent(match?.assetId)}`
```

Comment:
This feels really brittle to me. If we change how to route to asset pages, that will break this and it's not going to be obvious to a developer that this relies on a specific route format.